org.eclipse.dartboard/src/org/eclipse/dartboard/preference/DartSDKLocationFieldEditor.java

```diff
@@ -65,7 +66,7 @@ private boolean isValidDartSDK(String location) {
 		// InvalidPathException is thrown. In that case we can assume that the location
 		// entered is not a valid Dart SDK directory either.
 		try {
-			path = Paths.get(location).resolve("bin/dart");
+			path = Paths.get(location).resolve("bin" + File.separator + "dart");
```

Comment:
Still, it will fail on windows. There is no file with the name "dart".It should be dart.exe

Reply:
Ah, right. I missed that. sorry.